resources/default-services/jukebox-daemon.service

```diff
@@ -1,15 +1,17 @@
 [Unit]
 Description=Jukebox Daemon
 # pulseaudio.service is required during shut down of this service, as we play shutdown sound (and adjust the volume)
-After=network.target sound.target mpd.service pulseaudio.service
-Requires=mpd.service pulseaudio.service
+After=network.target sound.target
```

Comment:
Thanks for improving the Service configuration file. Could you please elaborate on these changes? 
We had severe problems with the service shutdown, which dead-locked with the PulseAudio service shutting down. Hence the PulseAudio requirement. `sound.target` seems the more generalized choice. Could you please test to shutdown the Jukebox from within the service (use the run_rpc_tool.py or the web interface). After reboot check the logs to confirm the shutdown process went though w/o errors and in about 1-2s. You need to look at the file app.log.1 (app.log being the log of the current run).
Why do you remove MPD? I am not sure how the Jukebox behaves when it cannot connect to MPD during it's initialization. It's a good point: it should be implemented to work w/o issues and try to re-connect when an MPD command is issued. Just not sure it has been tested yet.

Reply:
I just went through the service files. PulseAudio is started after sound.target and is wanted by default.target. We need to run AFTER pulseaudio is up (especially for the service shutdown in opposite order). We can test dropping the MPD requirement here, once we have adjusted the Jukebox code to be able to deal with missing MPD connection during start-up (by re-trying to connect to MPD on every issued MPD command). Currently, this is not the case and I presume this could break the system in unpredictable ways as boot behaviour could vary from machine to machine.

Reply:
thank you for the explanation - I fully agree. And yes systemd service files are not easy if you want to understand them fully.